storage/connect/CMakeLists.txt

```diff
@@ -443,12 +442,20 @@ ENDIF()
 IF(CONNECT_WITH_JDBC AND JAVA_FOUND AND JNI_FOUND)
   # TODO: Find how to compile and install the java wrapper classes
   # Find required libraries and include directories
+  INCLUDE(UseJava)
   SET (JAVA_SOURCES JdbcInterface.java)
-  add_jar(JdbcInterface ${JAVA_SOURCES})
+  add_jar(JdbcInterface ${JAVA_SOURCES} OUTPUT_DIR ${CMAKE_CURRENT_BINARY_DIR}/connect_jars)
+  get_target_property(_jarFile JdbcInterface JAR_FILE)
   INSTALL(FILES
-    ${CMAKE_CURRENT_SOURCE_DIR}/mysql-test/connect/std_data/JavaWrappers.jar
-    ${CMAKE_CURRENT_BINARY_DIR}/JdbcInterface.jar
-    DESTINATION ${INSTALL_MYSQLSHAREDIR} COMPONENT connect-engine)
+    ${_jarFile} DESTINATION ${INSTALL_MYSQLSHAREDIR} COMPONENT connect-engine-jdbc)
+  IF (RPM)
+    SET(CPACK_COMPONENTS_ALL ${CPACK_COMPONENTS_ALL} connect-engine-jdbc PARENT_SCOPE)
+    SET(CPACK_RPM_connect-engine-jdbc_PACKAGE_REQUIRES "java-11-openjdk-devel" PARENT_SCOPE)
```

Comment:
We can not use  `jre` instead of `jdk`, because we are compiling java source and need `jar`, that is part of `jdk` and not `jre`.
Why `*openjdk-devel` cannot stay as suggested `CPACK_RPM_connect-engine-jdbc_PACKAGE_REQUIRES` instead of suggestion `CPACK_RPM_BUILDREQUIRES` ? In description is showed that is recognized properly as requirement.

Reply:
`CPACK_RPM_connect-engine-jdbc_PACKAGE_REQUIRES` is the runtime depends which doesn't need `jar`, and other `jdk` tools, at runtime. It can be set to `JAVA_JVM_LIBRARY_DEP`. Do an error if `JAVA_JVM_LIBRARY_DEP` isn't set.

It cannot stay because it is excessive and out of sync with the deb.

The `FIND_FILE` above is sufficient for the `cmake/build_depends.cmake` called at the top level to populate `CPACK_RPM_BUILD_REQUIRES` with the jdk. Ensure to comment its purpose.